maven/jdk11/Dockerfile

```
@@ -4,11 +4,16 @@ FROM maven:3.6.3-jdk-11-slim
 
 RUN apt-get update && \
     apt-get install -y \
-        git \
+        git bash \
         libfontconfig1 \
         libfreetype6
```

```suggestion
        git bash \
        libfontconfig1 \
        libfreetype6 && \
    apt-get clean && \
    rm -rf /var/lib/apt/lists/*
```
The idea is to cleanup the apt-get's index right after installing the packages, to allow carrying outdated data into the image.
The reason of suggesting this change in this PR is to be homoegenous with the ruby image (https://github.com/jenkinsci/jnlp-agents/pull/23/files#diff-4b62900532c579e3c7c952ed7e68a5614cfb8f35adf4640c4523c54fd3e29271R7).